content/guides/farmlands-pos-integration.md

```diff
@@ -85,257 +85,246 @@ sequenceDiagram
 
 1. The Cardholder presents their Farmlands card for the POS to scan.
 2. (Optional) The POS decodes the Scanned Barcode to confirm that it is valid and applies Farmlands discounts.
-    - Example API Request [[API Reference](https://docs.centrapay.com/api/scanned-codes#decode-scanned-code)]
-
-        ```bash
-        curl -X POST https://service.centrapay.com/api/decode \
-          -H "Authorization: $jwt" \
-          -H "Content-Type: application/json" \
-          -d '{
-            "code": "123456789",
-            "scannedBy": "merchant",
-            "merchantConfigId": "5ee168e8597be5002af7b454"
-          }'
-        ```
-
-        Example API Response Payload
-
-        ```bash
-        {
-          "code": "123456789",
-          "scannedBy": "merchant",
-          "merchantConfigId": "5ee168e8597be5002af7b454",
-          "provider": "farmlands",
-          "displayName": "Farmlands Card"
-        }
-        ```
+- [[API Reference](https://docs.centrapay.com/api/scanned-codes#decode-scanned-code)]
+
+    ```bash [Example API Request]
+    curl -X POST https://service.centrapay.com/api/decode \
+      -H "Authorization: $jwt" \
+      -H "Content-Type: application/json" \
+      -d '{
+        "code": "123456789",
+        "scannedBy": "merchant",
+        "merchantConfigId": "5ee168e8597be5002af7b454"
+      }'
+    ```
+
+    ```bash [Example API Response Payload]
```

Comment:
Suggest we rename these captions:
+ Example API Request -> Request
+ Example API Response -> Response

Reply:
@tom-blake done 👍